tasks/main.yml

```diff
@@ -98,5 +98,5 @@
     - name: tlwebaccess
       enable: "{{ thinlinc_webaccess_enabled }}"
   loop_control:
-    loop_item: service
+    loop_var: service
```

Comment:
this should be combined with the commit that introduced this typo

Reply:
Sorry, I'm used to working on PRs where all commits are squashed at merge. Should I squash all commits beforehand?

Reply:
You should squash those commits who belongs together, like "fixed typo". Let's try to keep a clean & neat commit history.

Reply:
Done!